src/entity/EnemyShip.java

```diff
@@ -136,11 +146,19 @@ public final void update() {
 	/**
 	 * Destroys the ship, causing an explosion.
 	 */
-	public final void destroy() {
-		this.isDestroyed = true;
-		this.spriteType = SpriteType.Explosion;
+
+	public final void destroy() {      //적 함선 체력에 따라 파괴여부 결정
+		if(this.health == 0){
+			this.isDestroyed = true;
+			this.spriteType = SpriteType.Explosion;
+		}else{
+			this.health--;
+		}
```

Comment:
How about adopting the current approach of introducing a health system to enemy ships, but keeping the destroy() method that can be destroyed at once? The Bomb and Line Bomb developed by the Item team are planned to destroy enemies in a certain range. I can modify the code to change it to deal area damage instead of destroying them, but it seems better to go with the original plan.